### DISCUSSION TITLE
how can i get more info about the rocksmq in Standalone

### DISCUSSION THREAD

**Discussion by huangxueliang** [milvus-io/milvus#37662] [Show and tell]
how can i get more info about the rocksmq in Standalone,such as login the rocksmq and execuete command get some info.

**Comment by yhmo** [comment]
Rocksmq is a message queue based on rocksdb, managed internally by milvus, and invisible to users. Users no need to know how it works, and can do nothing to the internal rocksdb.

**Reply by huangxueliang** [reply]
![image](https://github.com/user-attachments/assets/4726dabb-aff2-4a03-9dca-e986935524d6)
the rdb's data can be delete by manual?

**Reply by yhmo** [reply]
You might lost data if you delete any internal managed data folders in milvus.
